node/res/eden-testing.json

```diff
@@ -18,7 +18,9 @@
   "protocolId": "nodl-testing-0510",
   "properties": {
     "tokenDecimals": 11,
-    "tokenSymbol": "notNODL"
+    "tokenSymbol": "notNODL",
+    "ss58Format": 37
+
```

Comment:
Looks good to me. If there's a blank line here (as it appears) we might want to remove it, and consider adding a JSON formatter in the future.